internal/aws/awss3configsource/config.go

```diff
@@ -0,0 +1,32 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awss3configsource
+
+import (
+	splunkprovider "github.com/signalfx/splunk-otel-collector"
```

Comment:
It seems quite odd for the upstream collector to depend on a downstream distro, it's like a circular dependency

Reply:
Splunk is currently submitting a string of PRs into Collector - the plan is to have this merged in - in fact in the go.mod file I replace this reference with a local reference to the config provider folder